dns/Makefile

```diff
@@ -13,22 +13,40 @@
 # limitations under the License.
 
 SHELL := /usr/bin/env bash
-CANARY_ZONES := canary.k8s.io. canary.kubernetes.io. canary.x-k8s.io. canary.k8s-e2e.com.
-PROD_ZONES := k8s.io. kubernetes.io. x-k8s.io. canary.k8s-e2e.com.
+CANARY_ZONES := canary.k8s.io. canary.kubernetes.io. canary.x-k8s.io. canary.k8s-e2e.com. canary.k8s.dev. canary.kubernetes.dev.
+PROD_ZONES := k8s.io. kubernetes.io. x-k8s.io. k8s-e2e.com. k8s.dev. kubernetes.dev.
```

Comment:
If you flip these lines, put PROD first, then you can do

```
CANARY_ZONES := $(foreach z,$(PROD_ZONES),canary.$(z))
```

(untested, may have typos)